web/styles/zulip.css

```diff
@@ -172,6 +172,9 @@ body {
     */
     --max-unexpanded-compose-height: 40vh;
 
+    /* Line height of the message buttons in compose box. */
```

Comment:
I think it could be worth briefly mentioning here that it's necessary to control this value uniformly and precisely to avoid issues with layout shifts originating with non-Latin characters this area.